0000-template.md

```diff
@@ -1,51 +1,48 @@
+- Title: (fill me in with a human-focussed title for the RFC)
 - Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- RFC PR: [#0000](https://github.com/rust-lang/rfcs/pull/0000)
 - Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
```

Comment:
I have been tempted to suggest renaming "Rust Issue" to something like "Tracking Issue". That way it isn't quite so unclear about its purpose, and avoids confusion whether it is intended to live in the "rust" repo.